.tekton/run-acceptance-test.yaml

```diff
@@ -0,0 +1,88 @@
+    pipelinesascode.tekton.dev/task-28: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/get-component-versions.yaml"
+    pipelinesascode.tekton.dev/task-29: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upload-results-to-polarion.yaml"
+    pipelinesascode.tekton.dev/task-30: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/get-operator-component-version-unofficial-builds.yaml"
+    pipelinesascode.tekton.dev/task-31: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upload-test-cases-to-polarion.yaml"
+    pipelinesascode.tekton.dev/task-32: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/get-operator-files-unofficial-builds.yaml"
+    pipelinesascode.tekton.dev/task-33: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-operator-tests.yaml"
+    pipelinesascode.tekton.dev/task-34: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/install-cluster.yaml"
+    pipelinesascode.tekton.dev/task-35: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-pac-tests.yaml"
+    pipelinesascode.tekton.dev/task-36: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/install-serverless-operator.yaml"
+    pipelinesascode.tekton.dev/task-37: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-pipelines-tests.yaml"
+    pipelinesascode.tekton.dev/task-38: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/mini-must-gather.yaml"
+    pipelinesascode.tekton.dev/task-39: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-triggers-tests.yaml"
+    pipelinesascode.tekton.dev/task-40: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/mirror-and-tag-imagestreams.yaml"
+    pipelinesascode.tekton.dev/task-41: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/verify-binaries-and-sha256sum.yaml"
+    pipelinesascode.tekton.dev/task-42: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/mirror-operator-images.yaml"
+    pipelinesascode.tekton.dev/task-43: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/verify-binaries-on-mac.yaml"
+    pipelinesascode.tekton.dev/task-44: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/release-tests.yaml"
+    pipelinesascode.tekton.dev/task-45: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/whitelist-domains.yaml"
+    pipelinesascode.tekton.dev/task-46: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/run-tfa-classifire.yaml"
+    pipelinesascode.tekton.dev/task-47: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/cleanup-cloud-resources.yaml"
+    pipelinesascode.tekton.dev/task-48: "git-clone"
```

Comment:
We use cluster resolver for git-clone, is this necessary? If yes, are you sure that it uses the correct task?